misk/src/main/kotlin/misk/metrics/Metrics.kt

```diff
@@ -34,17 +34,12 @@ class Metrics @Inject internal constructor(val registry: CollectorRegistry) {
     return Gauge.build(name, help).labelNames(*labelNames.toTypedArray()).register(registry)
   }
 
-  fun summary(name: String, help: String = "", labelNames: List<String> = listOf()): Summary {
-    return Summary.build(name, help).labelNames(*labelNames.toTypedArray()).register(registry)
-  }
-
   fun histogram(
     name: String,
     help: String = "",
-    labelNames: List<String> = listOf(),
-    buckets: DoubleArray? = null
+    labelNames: List<String> = listOf()
```

Comment:
The caller will also need to decide which quantiles will be ultimately emitted as metric, so also `quantiles: DoubleList = DefaultQuantiles`